src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -1,140 +1,179 @@
 import json
 import logging
+from typing import Dict, List
 
 import d3a.constants
-from d3a.d3a_core.exceptions import InvalidBidOfferPair
+from d3a.d3a_core.exceptions import (
+    InvalidBidOfferPairException, OfferNotFoundException,
+    BidNotFoundException, MycoValidationException)
 from d3a.d3a_core.redis_connections.redis_area_market_communicator import ResettableCommunicator
-from d3a.models.market import validate_authentic_bid_offer_pair
-from d3a.models.market.market_structures import (
-    BidOfferMatch, offer_or_bid_from_json_string)
+from d3a.models.market import Market
+from d3a.models.market.market_structures import BidOfferMatch
 
 from d3a.models.myco_matcher.base_matcher import BaseMatcher
 
 
 class ExternalMatcher(BaseMatcher):
+    """Class responsible for external bids / offers matching."""
     def __init__(self):
         super().__init__()
         self.simulation_id = d3a.constants.COLLABORATION_ID
         self.myco_ext_conn = None
-        self.channel_prefix = f"external-myco/{self.simulation_id}/"
+        self.channel_prefix = f"external-myco/{self.simulation_id}"
+        self.response_channel = f"{self.channel_prefix}/response"
+        self.events_channel = f"{self.response_channel}/events/"
```

Comment:
It seems like most of these variables are just internals and they should not be changed nor accessed from the outside. If this is the case, I would suggest to name them as "private" (e.g. `self._events_channel`) to make it clear.